src/core/server/config/deprecation/core_deprecations.ts

```diff
@@ -382,7 +382,7 @@ const quietLoggingDeprecation: ConfigDeprecation = (
       message: i18n.translate('core.deprecations.loggingQuiet.deprecationMessage', {
         defaultMessage:
           '"logging.quiet" has been deprecated and will be removed ' +
-          'in 8.0. Moving forward, you can use "logging.root.level:error" in your logging configuration. ',
```

Comment:
These added spaces were just typos that I caught when converting the snapshot tests.

Reply:
ya, I think my VSCode setup needs some TLC